ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -1617,7 +1617,8 @@ def resolve_record(nameserver, query, rtype="SOA", retry=True, timeout=100):
     raise errors.DNSResolverError(exception=ValueError("Record not found"))
 
 
-def ipa_backup(host, disable_role_check=False, raiseonerr=True):
+def ipa_backup(host, disable_role_check=False, data_only=False,
+               raiseonerr=True):
```

Comment:
The method does not take into account the data_only param. A block similar to
```
    if data_only:
        cmd.append('--data')
```
needs to be added before the call to `host.run_command`.

Reply:
Looks like I got ahead of myself and fixed the calls to ipa_backup before adding the option. Fixed now.